README.md

```diff
@@ -13,9 +13,6 @@
 
 The Confidential Consortium Framework (CCF) is an open-source framework for building a new category of secure, highly available,
 and performant applications that focus on multi-party compute and data.
-CCF can enable high-scale, confidential networks that meet key enterprise requirements — providing a means to accelerate production and enterprise adoption of consortium based blockchain and multi-party compute technology.
```

Comment:
Can we update in on the Concepts page as well? https://microsoft.github.io/CCF/main/overview/concepts.html#ccf-in-a-hundred-words

Reply:
Yes, done.